.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.7.7
```

Comment:
Since this is open source, I would actually add a commit before this commit that supports ruby 3.0.x
Otherwise we have a weird unsupported gap